Converters/ArcGIS/Speckle.Converters.ArcGIS3/ToSpeckle/Raw/GisFeatureToSpeckleConverter.cs

```diff
@@ -2,114 +2,175 @@
 using Speckle.Converters.ArcGIS3.Utils;
 using Speckle.Converters.Common;
 using Speckle.Converters.Common.Objects;
+using Speckle.Objects;
+using Speckle.Objects.Geometry;
+using Speckle.Objects.GIS;
 using Speckle.Sdk.Models;
 
 namespace Speckle.Converters.ArcGIS3.ToSpeckle.Raw;
 
-public class GisFeatureToSpeckleConverter : ITypedConverter<Row, SGIS.GisFeature>
+public class GisFeatureToSpeckleConverter : ITypedConverter<Row, IGisFeature>
```

Comment:
This needs to reflect in VectorLayerToSpeckle converter: 
![image](https://github.com/user-attachments/assets/69e224b8-16f1-40ef-9b74-d186e4e7dbb4)